### DISCUSSION TITLE
Mirror list update request:Alibaba Cloud

### DISCUSSION THREAD

**Discussion by hjadb** [termux/termux-packages#9345] [General]
Apply to add Alibaba cloud mirror on this page.
https://github.com/termux/termux-packages/wiki/Mirrors.

Please refer to the following information.
1. Country: CN
2. Site Name: Alibaba Cloud
3. Mirror Name: mirrors.aliyun.com
4. Categories: https (https://mirrors.aliyun.com/termux/), http(http://mirrors.aliyun.com/termux/)
5. Bandwidth: 10000
6. Internet2: No
7. Email: ali-yum@alibaba-inc.com

Because this is neither "package request" nor "error report", I post it here.

Thanks!

**Comment by Grimler91** [comment]
Hey, sorry, haven't seen this thread before. Thanks for the info, is it your mirror, or who/what should I put as "responsible"? For the other mirrors in [the list](https://github.com/termux/termux-packages/wiki/Info-about-mirrors) we have written something like "Mirrors by Grimler" or "Mirrors by the Tsinghua University TUNA Association". Should I put "Mirrors by @hjadb"?

**Reply by 2096779623** [reply]
No, don't do this, you just need to set the mirror name to "Alibaba Cloud".😉

**Reply by Grimler91** [reply]
Is it sponsored/provided in any way by Alibaba, or does it just happen to be hosted on "Alibaba cloud"?

**Reply by 2096779623** [reply]
It is provided by Alibaba Cloud or free.

**Reply by agnostic-apollo** [reply]
Whose managing it?

There could be other users who host on Alibaba cloud, so can't name them all or just above one as `Alibaba Cloud`.

And ali baba host can change without changing the url, so it should be named after the manager. Check https://github.com/termux/termux-packages/wiki/Info-about-mirrors

**Reply by 2096779623** [reply]
Alibaba Cloud, like [Huawei Cloud Mirror](https://mirrors.huaweicloud.com/), is run by enterprises.

**Reply by agnostic-apollo** [reply]
Is Alibaba Cloud internal team managing the mirror themselves or is some third party user/party who is using the Alibaba cloud account to host the mirror?

**Reply by 2096779623** [reply]
Is Alibaba Cloud internal team managing the mirror.

**Reply by agnostic-apollo** [reply]
More info at

https://mirrors.aliyun.com/about

https://mirrors-aliyun-com.translate.goog/about?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

>Alibaba Cloud Mirror is supported by the Basic Systems Group of the Alibaba Technical Support Department.

This can be set to be provided by [`Alibaba Open Source Mirror Site`](https://mirrors.aliyun.com)

**Reply by 2096779623** [reply]
> 

This is great!

**Comment by Grimler91** [comment]
Has now been added to termux-change-repo and the mirror wiki page
